integration-tests/main/src/test/resources/image-metrics-test.properties

```diff
@@ -0,0 +1,23 @@
+image_details.total_bytes=138778008
+image_details.total_bytes.tolerance=3
+# TODO: Switch to using analysis_results.types.total key once we drop support for GraalVM 22.3.0
```

Comment:
This comment seems outdated? Did you mean `analysis_results.types.reachable`?

Reply:
Correct